docs/s3fs-fuse-with-minio.md

```diff
@@ -0,0 +1,61 @@
+# s3fs-fuse with Minio Server [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+s3fs allows Linux and Mac OS X to mount an S3 bucket via FUSE. Please note that you will not be able to create directories with s3fs because Minio does not support creating folders.
+
+In this recipe we will learn how to configure and use s3fs to mount a bucket from the Minio Server and copy data to it. 
+## 1. Prerequisites
+
+Install Minio Server from [here](http://docs.minio.io/docs/minio).
+
+## 2. Installation
+
+Install `s3fs-fuse` from <https://github.com/s3fs-fuse/s3fs-fuse>.
+
+Create a bucket on the Minio Server to use with this cookbook.
+
+## 3. Configuration
+
+Before you run s3fs, you will need to save your S3 credentials in a file that will be used later in this tutorial. In the command below, replace access_key and secret_key with your actual Minio credentials. 
```

Comment:
``Before you run s3fs, you will need to save your Minio credentials`` - Minio instead of S3 to avoid confusion.